framework/source/class/qx/test/Interface.js

```diff
@@ -365,7 +363,7 @@ qx.Class.define("qx.test.Interface",
             members :
             {
               getValue : function() {},
-              setValue : function(value) {}
```

Comment:
@pcdummy does the generator linter really complain here? A few lines below there is still an empty function with parameter value : https://github.com/pcdummy/qooxdoo/blob/297d25359f857c4d4396fbbe5a69e7f60d4bbd36/framework/source/class/qx/test/Interface.js#L401